doc/display.xml

```diff
@@ -288,6 +303,30 @@ gap> FileString("t3.dot", DotString(S));
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DotStringDigraph">
+  <ManSection>
+    <Oper Name="DotString" Arg="digraph" Label="for a Cayley digraph"/>
+    <Returns>A string.</Returns>
+    <Description>
+      If <A>digraph</A> is a <Ref Oper="Digraph" BookName="digraphs"/> in the
+      category <Ref Filt="IsCayleyDigraph" BookName="digraphs"/>, then
+      <C>DotString</C> returns a graphical representation of <A>digraph</A>. 
```

Comment:
I think you should be more descriptive about what the representation actually shows.